### DISCUSSION TITLE
Question about versioning

### DISCUSSION THREAD

**Discussion by pat0x0** [searxng/searxng#1947] [Q&A] [answered]
Hello! I see that they no longer publish versions in the form of tags here on github, and only do it in tags almost daily in Docker. There are no longer stable versions and they migrated to a rolling release model? This puzzles me, more than anything because I would like to have a public instance. Greetings.

**Comment by return42** [comment, marked as answer]
From the docs: [SearXNG is a rolling release; each commit to the master branch is a release.](https://docs.searxng.org/admin/update-searxng.html)

- https://github.com/searxng/searxng/pull/917
- https://github.com/searxng/searxng/issues/915
- https://github.com/searxng/searxng/issues/633
- https://github.com/searxng/searxng/issues/1835#issuecomment-1267480594

**Comment by pat0x0** [comment]
Sorry! I got tired of looking for the answer in the Q&A of the repository and it ended up being found on the website and in the issues... 😆 Thanks!
